source/IOSResolver/src/IOSResolver.cs

```diff
@@ -295,6 +300,7 @@ private class IOSXmlDependencies : XmlDependencies {
         ///   <iosPods>
         ///     <iosPod name="name"
         ///             path="pathToLocal"
+        ///             podspec="pathToLocalPodSpecJSON"
```

Comment:
I think you can specify a url here as well
https://guides.cocoapods.org/syntax/podfile.html#pod

Perhaps use
```
/// podspec="pathToPodSpec"
```